src/components/Map/components/LegendGroupLabel.tsx

```diff
@@ -9,6 +9,7 @@ import {
 export type TypeGroupLegend = {
   id: string;
   subLegend?: string;
+  subLegendTxt?: string;
```

Comment:
What's the difference between both ?

Reply:
- subLegend is a ref to an 'React Functional Component' (For advanced render, like the Box under 'Bâtiments chauffés au gaz collectif' or 'Bâtiments chauffées au fioul collectif')
- subLegendTxt is just a string